packages/terra-list/src/ListItem.jsx

```diff
@@ -94,6 +100,10 @@ const ListItem = ({
     attrSpread['data-item-show-focus'] = 'true';
     attrSpread.onBlur = ListUtils.wrappedEventCallback(onBlur, event => event.currentTarget.setAttribute('data-item-show-focus', 'true'));
     attrSpread.onMouseDown = ListUtils.wrappedEventCallback(onMouseDown, event => event.currentTarget.setAttribute('data-item-show-focus', 'false'));
+  } else if (isRoleListbox) {
+    attrSpread.role = 'option';
+    attrSpread['aria-selected'] = isSelected;
+    attrSpread['aria-disabled'] = 'true';
```

Comment:
Why would we assign aria-disabled if an item is selected? I would expect disabled to be applied if the item was disabled from receiving a click event